.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -78,6 +78,12 @@ Identify/create the bundles that we will release for Kata and enclave-cc.
 
 ### Test Release with Operator
 
+    NOTE: There is a release-helper script for these steps in the operator
+    repo [here](https://github.com/confidential-containers/operator/tree/main/hack/release-helper.sh).
+    Before using it, however, you should sanity check that the list of
+    prereq- and kata-related yaml files that it modifies are correct (see
```

Comment:
Do you still want this PR @portersrc ?
Looks like your dependent PR is merged. But the reference here to a yaml list wasn't part of that PR so these 2 lines would need to go.

Reply:
Hey @dcmiddle thanks for the bump! I think it's OK to take the PR but without the comment about those yaml files now. Just pushed an update